### PR TITLE
Add decimal data in the primitive type in the specification of the Delta Transaction Protocol

### PR DESCRIPTION
**Pull request by fvaleye**
The Decimal primitive type is missing in the Delta Transaction Protocol specification.

### CODE REVIEW

File: PROTOCOL.md [hunk 1]
```diff
@@ -504,6 +504,7 @@ short| 2-byte signed integer numbers. Range: -32768 to 32767
 byte| 1-byte signed integer number. Range: -128 to 127
 float| 4-byte single-precision floating-point numbers
 double| 8-byte double-precision floating-point numbers
+decimal| arbitrary-precision signed decimal numbers. Backed internally by java.math.BigDecimal. A BigDecimal consists of an arbitrary precision integer unscaled value and a 32-bit integer scale.
```
Comment (liwensun) [inline PROTOCOL.md:507]: How about this:

```
signed decimal number with fixed precision (maximum number of digits) and scale (number of digits on right side of dot). The precision and scale can be up to 38.
```

**Review by liwensun — APPROVED** [review]
LGTM. thanks for fixing it.

**Comment by houqp** [comment]
@liwensun could you let us know what's left to be done to get this merged?

**Comment by liwensun** [comment]
@houqp @fvaleye Sorry this PR got missed. I've included in the next batch merge which should happen within the next couple days.

**Comment by houqp** [comment]
Thanks @liwensun !

**Pull request closed without merging**